rules/windows/defense_evasion_suspicious_powershell_imgload.toml

```diff
@@ -0,0 +1,90 @@
+[metadata]
+creation_date = "2020/11/17"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/11/17"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies the PowerShell engine being invoked by unexpected processes. Rather than executing PowerShell functionality
+with powershell.exe, some attackers do this to operate more stealthily.
+"""
+from = "now-9m"
+index = ["logs-endpoint.events.*", "winlogbeat-*"]
+language = "eql"
+license = "Elastic License"
+name = "Suspicious PowerShell Engine ImageLoad"
+risk_score = 43
+rule_id = "852c1f19-68e8-43a6-9dce-340771fe1be3"
+severity = "medium"
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Defense Evasion"]
+type = "eql"
+
+query = '''
+library where file.name == "System.Management.Automation.ni.dll" and
```

Comment:
Can we add the other related PowerShell engine DLL (System.Management.Automation.dll) too?  I know when I did previous testing with SharpPick, we record the image load, but it wasn't part of the rule. 

![image](https://user-images.githubusercontent.com/57736958/99456379-ed178080-28ee-11eb-82bf-e1405c7a3e89.png)


```suggestion
library where file.name in ("System.Management.Automation.ni.dll", "System.Management.Automation.dll") and
```

Reply:
good catch